app/templates/_package.json

```diff
@@ -9,28 +9,28 @@
     "url": "<%= authorURL %>"
   },
   "scripts": {
+    "setup": "wp core download && npm install && bower install && npm run install-wp-plugins",
     "start": "grunt serve",
     "build": "grunt build",
-    "deploy": "git push && git push staging development",
+    "deploy": "git push origin development",
```

Comment:
I'm wondering if we even need the deploy command at all. At least in the generator.